### ISSUE TITLE
dependencies are broken.

### ISSUE THREAD

**Issue by remco-remco** [wavesplatform/go-lib-crypto#1]
Some import dependencies are broken.

"github.com/agl/ed25519" should be replaced by -> "golang.org/x/crypto/ed25519"
"github.com/agl/ed25519/edwards25519" should be replaced by -> ???

**Comment by alexeykiselev** [comment]
It's impossible to replace `"github.com/agl/ed25519"` with `"golang.org/x/crypto/ed25519"` because while moving agl's package to `x/crypto` they placed the `edwards25519` package into `internal` folder and made it inaccessible.
Better to use cryptography from `"github.com/wavesplatform/gowaves/pkg/crypto"`.

**Comment by remco-remco** [comment]
> It's impossible to replace `"github.com/agl/ed25519"` with `"golang.org/x/crypto/ed25519"` because while moving agl's package to `x/crypto` they placed the `edwards25519` package into `internal` folder and made it inaccessible.
> Better to use cryptography from `"github.com/wavesplatform/gowaves/pkg/crypto"`.

Hi thanks for the reply, does it mean that this package is not maintained anymore?
edwards25519 can be made accessible again by manually moving it outside of the internal directory and adjusting the import path accordingly. (sorry but i have no idea yet why it was made inaccessible, there would be of course an obvious reason for it but i have no idea why at this moment still searching for that reason.)
Or do you know if there is a non "internal" x/crypto  equivalent replacement for it?
Thanks! I will also look into "github.com/wavesplatform/gowaves/pkg/crypto", thanks for the hint!

**Comment by alexeykiselev** [comment]
> Hi thanks for the reply, does it mean that this package is not maintained anymore?

No, not at all. But this package was intended to support the common API of all client libraries in Waves ecosystem. That's why it's quite simple and may be slightly outdated.

> edwards25519 can be made accessible again by manually moving it outside of the internal directory and adjusting the import path accordingly. (sorry but i have no idea yet why it was made inaccessible, there would be of course an obvious reason for it but i have no idea why at this moment still searching for that reason.)

We did exactly so in the `"github.com/wavesplatform/gowaves/pkg/crypto"` by coping the code from the standard library.
